### DISCUSSION TITLE
How to disable UseCDN on running instances.

### DISCUSSION THREAD

**Discussion by weirdyang** [OrchardCMS/OrchardCore#9378] [Show and tell]
Today a colleague enabled the UseCDN setting on a client which has no access to the CDN.

We had disabled the UseCDN setting using a step in the setup recipe:

```
        {
            "name": "settings",
            "UseCdn": false,
            "HomeRoute": {
                "Action": "Index",
                "Controller": "Admin",
                "Area": "OrchardCore.Admin"
            },
		}
```

To disable it again without re-running the recipe:

1. navigate to `https://localhost:5001/Admin/Settings/general`
2. You will be unable to go to the `Resources` tab as `jquery` isn't loaded
3. Right click -> Inspect page
4. Open console, type:

```
const checkBox = document.getElementById('ISite_UseCdn');
checkBox.value = 'false';
```
5. Click `Save`
6. The UseCDN feature will now be disabled.

To do this via a recipe:

```
{
    "name": "UseCdn to False",
    "displayName": "Set Use CDN to FALSE",
    "description": "Disable use of CDN",
    "author": "weirdyang",
    "website": "https://weirdyang.co",
    "version": "2.0",
    "issetuprecipe": "false",
    "categories": [ "settings" ],
    "tags": [ "settings", "default" ],
    "steps": [
        {
            "name": "settings",
            "UseCdn": false,
}
    ]
}
```

Hope someone finds this useful!

**Comment by jtkech** [comment]
@weirdyang Thanks for the tip 👍
